lib/gemika/matrix.rb

```diff
@@ -79,6 +138,9 @@ def each(&block)
         gemfile = row.gemfile
         if row.compatible_with_ruby?(current_ruby)
           @compatible_count += 1
+
+          @aliased_rubys[current_ruby] = row.ruby
```

Comment:
`@aliased_rubys` is an odd name in the case that row.ruby is just the current ruby without an alias.
You could, however, use `... if row.ruby != current_ruby` and then get rid of the `.select` in `print_aliases`